packages/microeditor-envelope/src/__tests__/AppFormerGwtApi.test.ts

```diff
@@ -0,0 +1,33 @@
+import { AppFormerGwtApi } from '../gwt/AppFormerGwtApi';
+import { DummyEditor } from './DummyEditor';
+import * as AppFormer from "appformer-js-core";
+import { GwtEditor } from '../gwt/GwtEditor';
+
+class DummyGWTEditor {
+  private wrappedEditor: AppFormer.Editor;
+
+  constructor (wrappedEditor: AppFormer.Editor) {
+    this.wrappedEditor = wrappedEditor;
+  }
+  public get(): AppFormer.Editor {
+    return this.wrappedEditor;
+  }
+}
+
+const dummyEditor: DummyEditor = new DummyEditor();
+const dummyGWTEditor: DummyGWTEditor = new DummyGWTEditor(dummyEditor);
+const editorId:string = "dummy editor";
+const appFormerGwtApi: AppFormerGwtApi = new AppFormerGwtApi();
+
+window.gwtEditorBeans = new Map<string, { get(): GwtEditor }>();
+window.gwtEditorBeans.set(editorId, dummyGWTEditor);
+
+describe("AppFormerGwtApi", () => {
+  test("get existing editor", () => {
+    const possibleEditor = appFormerGwtApi.getEditor(editorId);
+    expect(possibleEditor).toBeTruthy();
+  });
+  test("get non existing editor", () => {
+    expect(() => appFormerGwtApi.getEditor("X")).toThrowError("GwtEditor with id 'X' was not found");
```

Comment:
I think that just checking for an error is enough, we don't want to duplicate the error message on the tests.